src/amo/components/SearchResult/index.js

```diff
@@ -119,7 +122,9 @@ export class SearchResultBase extends React.Component<InternalProps> {
 
     // Sets classes to handle fallback if theme preview is not available.
     const iconWrapperClassnames = makeClassName('SearchResult-icon-wrapper', {
-      'SearchResult-icon-wrapper--no-theme-image': isTheme && imageURL === null,
```

Comment:
Fun fact: `isTheme` is not a prop here, it is a function imported at the top of the file. In the end, the condition is `imageURL === null` because `isTheme` exists and therefore `isTheme &&` should always be `true` (I think).

Given that `imageURL` can only be `null` if `addon` is defined, I added `addon` here because the `useThemePlaceholder` prop is only useful when we do not have an add-on.